.golangci.yml

```diff
@@ -0,0 +1,7 @@
+# See https://golangci-lint.run/usage/configuration/#config-file
+
+linters:
+  # NOTE: `enable` _adds_ to the default linters!
+  # To see the currently enabled linters, run `golangci-lint linters`
+  enable:
+    - gocritic
```

Comment:
Did we loose `gofmt` check?

Reply:
No, it is among the default linters (try with `golangci-lint linters`), but thanks for asking!

Reply:
I am confused by `golangci-lint run --enable gofmt,gocritic ./...`. Why did we have it here before? By mistake?

Reply:
Because we didn't have before file `.golangi.yml` and because, honestly, I don't know. I _think_ it was because when I added golangci-lint, gofmt was not a default, but maybe I was wrong all the time...